### DISCUSSION TITLE
MERN Projects

### DISCUSSION THREAD

**Discussion by akash-coded** [akash-coded/mern#116] [Tasks]
In the comments, submit your project(s) by doing the following:

- Mention the name of your project as a markdown heading.

- Paste a zipped file of your project folder.

- Paste the link to the GitHub repositories of your project(s).

- The project folder should contain an export of the working database which is used by your project.

- And include screenshots of your main front-end pages inside the "screenshots" folder under your front-end directory.

**Comment by HARSH062001** [comment]
### NETFLIX HOME PAGE 

### ZIPFILE
Can't upload due to file size , the allowed limt is 25 MB 
### Link
https://github.com/HARSH062001/netflixhomepage.git
https://github.com/HARSH062001/netflixhomepage

### Screenshots
![1](https://user-images.githubusercontent.com/89510042/196460286-b5a0b8d8-fae6-47b2-957f-f282951afae6.png)
![2](https://user-images.githubusercontent.com/89510042/196460298-8a594181-30c8-44c9-8280-3f9a389a9d19.png)
![3](https://user-images.githubusercontent.com/89510042/196460303-9cfc2314-7150-4e1b-bf8b-941ce7288cee.png)
![4](https://user-images.githubusercontent.com/89510042/196460318-bd88472e-206d-45fb-bbcc-d0d3a02ffa4b.png)
![5](https://user-images.githubusercontent.com/89510042/196460325-6a2ec9ee-4d0d-4c3a-9f13-f68297aa74a2.png)
![6](https://user-images.githubusercontent.com/89510042/196460336-98fc1f54-a5e8-405c-b5bd-d904b4d91397.png)
![7](https://user-images.githubusercontent.com/89510042/196460343-9fa850d9-0310-4ce5-93a9-10258c1bfdce.png)
![8](https://user-images.githubusercontent.com/89510042/196460350-ed60abb5-ec99-4c53-a1fe-bbe30fe2eb13.png)

### NETFLIX

### ZIPFILE
[netflix sign up.zip](https://github.com/akash-coded/mern/files/9811636/netflix.sign.up.zip)

###LINK
https://github.com/HARSH062001/netflixlanding.git
https://github.com/HARSH062001/netflixlanding

### SCREENSHOT

![1](https://user-images.githubusercontent.com/89510042/196464794-b3b13c14-3278-460e-8ec9-57d8f8949d13.png)
![2](https://user-images.githubusercontent.com/89510042/196464784-551b05cd-4db6-49ea-926e-7e6b0c66c8d8.png)

### MY CV

### ZIP FILE
[CV.zip](https://github.com/akash-coded/mern/files/9811520/CV.zip)

### LINK 
https://github.com/HARSH062001/cv.git
https://github.com/HARSH062001/cv

### SCREENSHOT
![1](https://user-images.githubusercontent.com/89510042/196463499-fdcce6a6-b3fc-4d30-95f4-5ac9e6ecdb58.png)
![2](https://user-images.githubusercontent.com/89510042/196463507-b92bb74e-91c4-4f33-8658-75394b13525e.png)
![3](https://user-images.githubusercontent.com/89510042/196463512-e98de03d-11f8-4645-a43a-5fa258345ddd.png)
![4](https://user-images.githubusercontent.com/89510042/196463515-9cb6c6fc-3a9e-44ff-b02e-80971851face.png)
![5](https://user-images.githubusercontent.com/89510042/196463518-47c1063c-23bd-4d22-8408-1bcf31553608.png)

**Comment by tash68** [comment]
# ed-zomato

## zip file 
[ed-zomato-main.zip](https://github.com/akash-coded/mern/files/9814618/ed-zomato-main.zip)

## Git Repo Link 
https://github.com/tash68/ed-zomato.git

# the-tash-cafe

## zip file 
[super-int-project-main rmcd.zip](https://github.com/akash-coded/mern/files/9814793/super-int-project-main.rmcd.zip)

## Git Repo Link
https://github.com/tash68/super-int-project.git
